tests/fixtures/example_transformations/sum_operation/single/config.yaml

```diff
@@ -0,0 +1,8 @@
+sum_operation:
+  - class_name: Dataset
+    target_content:
+      object_path: file_summary.stats
+      property_name: size
```

Comment:
Just a nitpick and not relevant for testing, but something like `cumulative_size` would fit what's being stored better.

Reply:
Haha thanks, true, but I rather have simple slot names in the example cases.